omero/developers/Tables.rst

```diff
@@ -295,13 +295,49 @@ These are only an introduction to using OMERO.tables and do not show its full
 potential, see :ref:`tables-going-forward` for some inspiration.
 
 
+Data viewing in OMERO.web
+-------------------------
+
+OMERO.web can display table data in a separate tab (attachment link) or in the image context, given the following setup:
```

Comment:
Tables are displayed in a similar way in the Insight client too.

Reply:
Do you refer to the ROI-Tool?
Could this be a separate section? 
(Viewing the attachment from a Dataset or Project in OMERO.insight did not look very readable to me).

Reply:
Apologies - you're right. Insight implemented this for Screen/Plate/Well but never did it for Project/Dataset/Image.